File: assignment 34/P1.java
import java.util.*;
class Mangesh
{
			public boolean CheckNoArray(int arr[],int ino)
			{
				int icnt=0;
				boolean bflag=false;
				for (icnt=0;icnt<arr.length;icnt++ ) 
				{
					if(arr[icnt]==ino)
					{
						bflag=true;
						break;
					}
				}
				return bflag;
			}
	
}
class P1
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		int ilen=0,i=0,ival=0;
		boolean bret=false;
		System.out.println("how many no you want");
		ilen=sc.nextInt();
		int brr[]=new int[ilen];
		System.out.println("enter no to find");
		ival=sc.nextInt();
		System.out.println("enter no");
		for(i=0;i<ilen;i++)
		{
			brr[i]=sc.nextInt();
		}
		Mangesh mobj= new Mangesh();
		bret=mobj.CheckNoArray(brr,ival);
		if(bret==true)
		{
			System.out.println("no is present");
		}
		else
		{
			System.out.println("no is not present");
		}
	}
}

File: assignment 18/p1.c
#include<stdio.h>
#include<stdlib.h>
#include<stdbool.h>
bool chk(int Arr[],int isize,int ino)
{
int icnt=0;
for(icnt=0;icnt<isize;icnt++)
{
if(Arr[icnt]==ino)
{
break;
}
}
if(icnt==isize)
{
return false;
}
else
{
return true;
}
}
int main()
{
int icnt=0;bool bret=false;
int ilength=0,ival=0;
int *ptr=NULL;
printf("enter how many elements you want");
scanf("%d",&ilength);
printf("enter no that have to chk");
scanf("%d",&ival);
ptr=(int*)malloc(sizeof(int)*ilength);
printf("enter elements");
for(icnt=0;icnt<ilength;icnt++)
{
scanf("%d",&ptr[icnt]);
}
bret=chk(ptr,ilength,ival);
if(bret==true)
{
printf("no is present");
}
else
{
printf("no is not present");
}
free(ptr);
return 0;
}
